### DISCUSSION TITLE
What should I consider when I want to create a components library with JSS

### DISCUSSION THREAD

**Discussion by Shinkei** [cssinjs/jss#1512] [Q&A] [answered]
Hi guys I am creating a React components library (first time doing that) and I would like to use JSS for styling and themes.
I would like that my components have a specific font family, and for that I am using a theme.

There are 2 questions, 
1) where can I put the font for my components, because I don't have an index.html
2) so I have to wrap every component in the theme provider and export it? Or how can I use the theme provider for my components' library?

Any recommendation or example 

Thanks in advance

**Comment by kof** [comment, marked as answer]
1. you can put font in any styles declaration
2. ThemeProvider should wrap the entire tree

**Reply by Shinkei** [reply]
Thanks!
